WebRoot/js/zimbraAdmin/cos/model/ZaCos.js

```diff
@@ -637,6 +638,7 @@ ZaCos.myXModel = {
         {id:ZaCos.A_zimbraMaxPwdAge, type:_NUMBER_, ref:"attrs/"+ZaCos.A_zimbraMaxPwdAge, maxInclusive:2147483647, minInclusive:0},
         {id:ZaCos.A_zimbraEnforcePwdHistory, type:_NUMBER_, ref:"attrs/"+ZaCos.A_zimbraEnforcePwdHistory, maxInclusive:2147483647, minInclusive:0},
         {id:ZaCos.A_zimbraPasswordLocked, type:_ENUM_, choices:ZaModel.BOOLEAN_CHOICES, ref:"attrs/"+ZaCos.A_zimbraPasswordLocked},
+        {id:ZaCos.A_zimbraPasswordBlockCommonEnabled, type:_ENUM_, choices:ZaModel.BOOLEAN_CHOICES, ref:"attrs/"+ZaCos.A_zimbraPasswordBlockCommonEnabled},
```

Comment:
should this be _COS_ENUM_ as per https://github.com/Zimbra/zm-admin-console/pull/70/files#diff-6f3991f8de85af7e3fb7f112d8494239R1894 ?

Reply:
I just take reference from another such pref “A_zimbraPasswordLocked”.
I failed to understand the underlying logic for the same.
I am open for suggestion here.